WebRoot/css/zm.css

```diff
@@ -7279,4 +7279,32 @@ input[type="radio"] + label {
 
 .ZmWhiteBlackViewNumberUsed {
 	@WhiteBlackListViewItemsNumberUsed@
-}
\ No newline at end of file
+}
+
+.ZOptionsSectionMain .ZmPrefMailBlockMessage {
+	@PrefMailBlockMessageCell@
+}
+
+.ZOptionsLabelTop.ZmPrefMailComposeLabel {
+	@PrefMailComposeLabel@
+}
+
+.ZmPrefMailComposeContent .ZOptionsLabelNarrow {
+	@PrefMailComposeRowLabel@
+}
+
+.ZmPrefMailComposeContent .ZOptionsField {
+	@PrefMailComposeRowValueWrapper@
+}
+
+.ZmPrefMailComposeContent .ZOptionsField .ZSelect > table {
+	@PrefMailComposeRowSelectWidget@
+}
+
+.ZmPrefMailComposeContent .ZWidgetTable {
+	@PrefMailComposeRowColorPickerWidget@
```

Comment:
I guess here you are trying to target color picker widget. If so, please have more specific class. `.ZWidgetTable` class is very generic and has been present in most of the component.